apps/common/views.py

```diff
@@ -3,40 +3,15 @@
 from rest_framework.response import Response
 
 
-# Create your views here.
-
-
 class HealthView(GenericAPIView):
```

Comment:
Aici e posibil fara metoda `get_queryset`. Ea nu are sens ca nu se foloseste. Poti sa folosessti `views.APIView`. In loc de `GenericAPIView`.

Reply:
Done